### ISSUE TITLE
Skip the sms/email sheet after taptopay. 

### ISSUE THREAD

**Issue by flipbosch** [sumup/sumup-ios-sdk#147]
Definitely the most smooth tap to pay integer to on I have seen. Because it’s not app to app. Checkout flow is very fast. Unfortunately we still have  the sms/email screen after a payment. While we did disable the onsuccess screen. is this because it’s mandatory by Apple? Or is their a way to switch it of. We already have option to mail and sms the client receipt. So it’s not needed for us. 

Once again compliments!

**Comment by jadeburton-sumup** [comment]
Thanks for the kind feedback @flipbosch  :) 

Can you please paste the code you're using to set the properties on `SMPCheckoutRequest` before starting a checkout?

**Comment by flipbosch** [comment]
> You're assigning a value twice to `skipScreenOptions`; the second assignment overwrites the first assignment.
> 
> Can you try this instead of the two assigments:
> 
> ```
>         request.skipScreenOptions = [SkipScreenOptions.success, SkipScreenOptions.failed]
> ```

Ah, making sence! Sorry. Working now. Happy its possible. Since now checkout is so fast and smooth!
